### PR TITLE
fix: keep agents desktop layout at 200% zoom

### PR DESCRIPTION
**Pull request by jaaydenh**
Fixes layout issues on the agents empty state page.

1. At 200% zoom on a 1440 px desktop, the CSS viewport shrinks to
   720 px, which was below the previous `md:` breakpoint (768 px) and
   collapsed the page into the mobile stack. Switching the page shell
   and shell-level controls to the `sm:` breakpoint (640 px) keeps the
   sidebar and chat area side-by-side at common zoom levels while
   preserving the mobile stack for real phone viewports.
2. The empty state stays bottom-aligned on mobile and centered on the
   desktop branch, with tighter spacing so the chat input sits closer to
   the bottom of the screen at 200% zoom.
3. The inner stack gap shrinks from `gap-4` (16 px) to `gap-2` (8 px)
   and the footer paragraph drops its `mt-1`, tightening the space
   around the organization selector, the chat input, and the
   "Introductory access to Coder Agents through September 2026" line.
4. Sidebar header/footer controls, the page header, the chat top bar,
   and the plan-mode badge now use the same `sm:` desktop breakpoint as
   the page shell. A collapsed sidebar can be expanded again at 640 to
   767 px.

Dropdown full-width CSS (`@media (max-width: 767px)`) and the
`isBelowMdViewport` helper are intentionally left at 768 px. Those
govern dropdown UX rather than page layout, and the chat pane is still
narrow at 640 to 767 px after the sidebar is visible. The page is in
desktop mode in that range while dropdowns stay full-width.

Screenshot 2026-04-30 at 23 03 48

Screenshot 2026-04-30 at 23 03 57

Regression coverage in Storybook:

- `AgentsPageView.stories.tsx > EmptyStateZoom200Desktop` pins a new
  720 px Chromatic viewport and asserts the rendered layout is
  horizontal, the sidebar is left of the main panel, and the sidebar
  header/footer controls are visible.
- `AgentsPageView.stories.tsx > CollapsedSidebarZoom200Desktop` pins
  the same 720 px viewport and asserts the expand-sidebar control is
  visible when the sidebar is collapsed.
- `AgentCreateForm.stories.tsx > OrgPickerTightSpacing` measures the
  vertical gap between the org selector row and the chat-input
  composer and expects it to stay below 16 px.

---

Generated by Coder Agents.

### CODE REVIEW

**Review by jaaydenh — COMMENTED** [review]
Deep review complete. Eight reviewers ran in parallel (test-auditor, edge-case-analyst, contract-auditor, frontend-reviewer, structural-analyst, modernization-ts, modernization-react, style-reviewer).

The layout fix is sound: consistent breakpoint shift across all three components, TDD-verified with Chromatic visual snapshots plus class-level assertions in vitest, and no P0/P1 correctness issues.

One P2 and a handful of lower-severity nits cluster around test brittleness in the two new Storybook stories. 1 P2, 1 P3, 3 nits across 5 inline comments.

(Deep review orchestrated by Coder Agents. Author is welcome to treat all findings as optional.)

File: site/src/pages/AgentsPage/components/AgentCreateForm.stories.tsx [hunk 1]
```diff
@@ -368,6 +368,43 @@ export const WithOrganizationPicker: Story = {
...
+	},
+	play: async ({ canvasElement }) => {
+		const canvas = within(canvasElement);
+		const orgTrigger = await canvas.findByTestId("compact-org-selector");
+		const input = await canvas.findByTestId("chat-message-input");
+
+		// The composer is the rounded-border wrapper around the chat
+		// input. Measuring from its top includes the border instead of
+		// just the inner textarea.
+		const composer = input.closest(".rounded-2xl") as HTMLElement | null;
```
Comment (jaaydenh) [inline site/src/pages/AgentsPage/components/AgentCreateForm.stories.tsx]: **P2** The composer lookup via `.rounded-2xl` couples the spacing regression test to an implementation-detail class. *(Edge Case Analyst P2, Contract Auditor Obs, Structural Analyst P3, Modernization React Nit, Style Reviewer Nit)*

> Edge Case Analyst: If the ChatMessageInput renderer wraps the composer in a new container or removes the class, the test throws "Expected composer wrapper to exist" without detecting the spacing regression. The story is meant to guard spacing, but instead guards a specific DOM shape.

> Structural Analyst adds: If the rounded border styling moves to a different utility class (e.g., `rounded-xl`, `rounded-3xl`) or is replaced with inline styles, the test fails.

Five reviewers converged on the same suggestion: add a `data-testid="chat-composer"` (or similar) on the composer wrapper in `AgentChatInput.tsx` and look it up with `findByTestId` instead of `input.closest(".rounded-2xl")`. Same DOM element, same measurement, test survives future class churn. Cost: two lines (one attribute + one lookup). Worth doing now while the test is fresh.

File: site/src/pages/AgentsPage/AgentsPageView.stories.tsx [hunk 2]
```diff
@@ -457,6 +457,47 @@ export const WithToolbarEndContent: Story = {
...
+
+		// Walk up from the chat input to find the outer agents page
+		// container. It is the closest ancestor with flex-direction
+		// set at a responsive breakpoint. We then assert the exact
+		// Tailwind class so the test fails if somebody reintroduces
+		// the md: breakpoint that caused the 200% zoom regression.
+		let el: HTMLElement | null = input;
+		let outer: HTMLElement | null = null;
+		while (el && el !== canvasElement) {
+			if (el.className.includes("flex-row")) {
```
Comment (jaaydenh) [inline site/src/pages/AgentsPage/AgentsPageView.stories.tsx]: **P3** The `className.includes("flex-row")` walk-up matches any ancestor with that class, not specifically the outer agents page container. *(Test Auditor P3, Edge Case Analyst P3, Structural Analyst P3, Modernization React Nit, Style Reviewer Nit)*

> Edge Case Analyst: If any ancestor between the input and the intended outer container happens to use flex-row (e.g., a flex row wrapper on an error alert, a button container, a subcomponent), the loop breaks early and asserts the wrong element. The test would pass even if the main layout switched back to `md:` because the assertion lands on an unrelated flex-row element.

Five reviewers again converge. Same fix pattern: `data-testid="agents-page-outer"` on the `<div>` at `AgentsPageView.tsx:159`, then `canvas.getByTestId("agents-page-outer")`. The class-level assertion on `sm:flex-row` / not `md:flex-row` is the right regression guard, but getting to the right element should not rely on DOM traversal.

File: site/src/pages/AgentsPage/AgentsPageView.stories.tsx [hunk 3]
```diff
@@ -457,6 +457,47 @@ export const WithToolbarEndContent: Story = {
...
+	parameters: {
+		viewport: { defaultViewport: "desktopZoom200" },
+		chromatic: { viewports: [720] },
+	},
+	play: async ({ canvasElement }) => {
+		const canvas = within(canvasElement);
+		const input = await canvas.findByTestId("chat-message-input");
+
+		// Walk up from the chat input to find the outer agents page
+		// container. It is the closest ancestor with flex-direction
```
Comment (jaaydenh) [inline site/src/pages/AgentsPage/AgentsPageView.stories.tsx]: **Nit** The comment says "flex-direction set at a responsive breakpoint" but the code below checks the `flex-row` class string. *(Style Reviewer Nit)*

> Style Reviewer: The comment is technically misleading. `flex-row` is not "flex-direction set at a responsive breakpoint", it's a specific Tailwind class.

Tighten the wording to match the implementation, e.g. "closest ancestor with the `flex-row` Tailwind class". Low stakes, but the current phrasing would mislead a reader who then wonders where the computed-style check is.

File: site/.storybook/preview.tsx [hunk 4]
```diff
@@ -50,6 +50,20 @@ export const parameters: Parameters = {
...
+			// levels. The AgentsPage empty state flips to the mobile stack
+			// at the Tailwind sm: breakpoint (640 px), so 720 px must stay
+			// on the desktop branch.
+			desktopZoom200: {
+				name: "Desktop @ 200% zoom (720x450)",
+				styles: {
+					height: "450px",
+					width: "720px",
+				},
+				type: "desktop",
```
Comment (jaaydenh) [inline site/.storybook/preview.tsx]: **Obs** `desktopZoom200` is typed `"desktop"` despite its 720 px CSS width. *(Edge Case Analyst Obs)*

> Edge Case Analyst: If test infrastructure assumes desktop viewports are >= 768 px, the 720 px viewport may be miscategorized in visual regression reports or excluded from certain automated checks.

In Storybook's viewport addon the `type` field is mostly a UI affordance (groups the entry in the dropdown), so the current value is fine in practice. Worth flagging only because future Chromatic or custom viewport consumers might split on `type`. No action needed unless you want to future-proof.

File: site/src/pages/AgentsPage/AgentsPageView.tsx [hunk 5]
```diff
@@ -156,17 +156,17 @@ export const AgentsPageView: FC<AgentsPageViewProps> = ({
 	};
 
 	return (
-		<div className="flex h-full min-h-0 flex-col overflow-hidden bg-surface-primary md:flex-row">
+		<div className="flex h-full min-h-0 flex-col overflow-hidden bg-surface-primary sm:flex-row">
```
Comment (jaaydenh) [inline site/src/pages/AgentsPage/AgentsPageView.tsx]: **Obs** Page layout now uses `sm:` (640 px) while dropdown UX still keys off `isBelowMdViewport()` / `@media (max-width: 767px)`. *(Contract Auditor P1, Structural Analyst Obs)*

> Contract Auditor: A viewport at 700 px will appear to be in two different "modes" depending on which component you ask. The user sees AgentsPage layout: desktop mode (sidebar, chat side-by-side) [and] Chat input dropdowns: mobile mode (full-width).

> Structural Analyst adds: If someone later changes `isBelowMdViewport` to use a different threshold without reading the PR description, the UX will break.

The PR description already calls this out as an accepted short-term trade-off. Cross-check didn't find a correctness issue in the 640–767 px band (desktop layout plus full-width dropdowns is a reasonable combo), so downgrading Contract Auditor's P1 to Obs here. The concrete follow-up if you want it: drop a short comment near the `isBelowMdViewport()` call sites in `AgentChatInput.tsx` referencing this PR so a future reader knows the breakpoint split is deliberate.

**Comment by jaaydenh** [comment]
@codex review

**Comment by jaaydenh** [comment]
/coder-agents-review
@codex review

File: site/src/pages/AgentsPage/AgentsPageView.tsx [hunk 6]
```diff
@@ -156,17 +156,20 @@ export const AgentsPageView: FC<AgentsPageViewProps> = ({
...
 					agentId
-						? "hidden md:block shrink-0 h-[42dvh] min-h-[240px] border-b border-border-default"
+						? "hidden sm:block shrink-0 h-[42dvh] min-h-[240px] border-b border-border-default"
 						: isSettingsDetail || isAnalytics
-							? "hidden md:block shrink-0"
-							: "order-2 md:order-none flex-1 min-h-0 border-b border-border-default md:flex-none md:border-t-0 md:border-b-0",
-					isSidebarCollapsed && "md:hidden",
+							? "hidden sm:block shrink-0"
+							: "order-2 sm:order-none flex-1 min-h-0 border-b border-border-default sm:flex-none sm:border-t-0 sm:border-b-0",
+					isSidebarCollapsed && "sm:hidden",
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/AgentsPageView.tsx:171]: Fixed in 7d916ee3f. The expand controls in AgentPageHeader and ChatTopBar now use the same sm breakpoint as the sidebar collapse gate, and CollapsedSidebarZoom200Desktop asserts that the expand button is visible at the 720 px zoom viewport.

File: site/src/pages/AgentsPage/AgentsPageView.tsx [hunk 7]
```diff
@@ -156,17 +156,20 @@ export const AgentsPageView: FC<AgentsPageViewProps> = ({
 	};
 
 	return (
-		<div className="flex h-full min-h-0 flex-col overflow-hidden bg-surface-primary md:flex-row">
+		<div
+			data-testid="agents-page-outer"
+			className="flex h-full min-h-0 flex-col overflow-hidden bg-surface-primary sm:flex-row"
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/AgentsPageView.tsx:159]: Fixed in 7d916ee3f. The page shell, sidebar chrome, page header, chat top bar, and AgentChatInput plan-mode/chrome controls now use the sm desktop breakpoint consistently. I kept the full-width dropdown helper at md intentionally because the chat pane remains narrow at 640 to 767 px after the sidebar is visible, and updated the inline comment and PR description to call that out.

File: site/src/pages/AgentsPage/AgentsPageView.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/AgentsPageView.tsx:171]: Fixed in 7d916ee3f. The sidebar hide gate and both expand controls now align on sm, so a sidebar collapsed at full width can be recovered after resizing or zooming to the 640 to 767 px range. Added CollapsedSidebarZoom200Desktop coverage for this.

File: site/src/pages/AgentsPage/AgentsPageView.stories.tsx [hunk 8]
```diff
@@ -457,6 +457,28 @@ export const WithToolbarEndContent: Story = {
...
+		viewport: { defaultViewport: "desktopZoom200" },
+		chromatic: { viewports: [720] },
+	},
+	play: async ({ canvasElement }) => {
+		const canvas = within(canvasElement);
+		const outer = await canvas.findByTestId("agents-page-outer");
+
+		await waitFor(() => {
+			const styles = getComputedStyle(outer);
+			expect(styles.flexDirection).toBe("row");
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/AgentsPageView.stories.tsx]: Fixed in 7d916ee3f. EmptyStateZoom200Desktop now asserts rendered behavior beyond flex-direction: the sidebar is visible, the main panel is flex, the sidebar is positioned to the left of the main panel, and sidebar header/footer controls are visible. The collapsed-sidebar story covers expand control reachability.

File: site/src/pages/AgentsPage/components/AgentCreateForm.tsx [hunk 9]
```diff
@@ -412,8 +412,8 @@ export const AgentCreateForm: FC<AgentCreateFormProps> = ({
 
 	return (
 		<>
-			<div className="order-last flex min-h-0 flex-none items-end justify-center overflow-auto p-4 pb-4 md:order-none md:h-full md:flex-1 md:items-center md:pt-12">
-				<div className="mx-auto flex w-full max-w-3xl flex-col gap-4">
+			<div className="order-last flex min-h-0 flex-none items-end justify-center overflow-auto px-4 pb-4 sm:order-none sm:h-full sm:flex-1 sm:items-center sm:pb-4">
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/AgentCreateForm.tsx]: Fixed. I updated the PR description to match the actual behavior: bottom-aligned on mobile, centered on the desktop branch, with tighter spacing at the 200% zoom desktop viewport.

File: site/src/pages/AgentsPage/components/AgentsSkeletons.tsx [hunk 10]
```diff
@@ -29,11 +29,11 @@ function getRightPanelState(): { open: boolean; width: number } {
...
  */
 export const AgentsPageSkeleton: FC = () => (
-	<div className="flex h-full min-h-0 flex-col overflow-hidden bg-surface-primary md:flex-row">
-		<div className="order-2 md:order-none flex-1 min-h-0 border-t border-border-default md:flex-none md:border-t-0 md:h-full md:w-[320px] md:min-h-0 md:border-b-0">
+	<div className="flex h-full min-h-0 flex-col overflow-hidden bg-surface-primary sm:flex-row">
+		<div className="order-2 sm:order-none flex-1 min-h-0 border-t border-border-default sm:flex-none sm:border-t-0 sm:h-full sm:w-[320px] sm:min-h-0 sm:border-b-0">
 			<div className="relative flex h-full w-full min-h-0 border-0 border-r border-solid overflow-hidden">
 				<div className="absolute inset-0 flex flex-col">
-					<div className="hidden border-b border-border-default px-2 pb-3 pt-1.5 md:block">
+					<div className="hidden border-b border-border-default px-2 pb-3 pt-1.5 sm:block">
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/AgentsSkeletons.tsx:36]: Fixed in 7d916ee3f. The real AgentsSidebar header and footer now use sm:block, matching the skeleton and avoiding the 640 to 767 px loading-to-loaded layout shift.

File: site/src/pages/AgentsPage/components/AgentChatInput.tsx [hunk 11]
```diff
@@ -924,6 +925,12 @@ export const AgentChatInput: FC<AgentChatInputProps> = ({
 										)}
 										{workspaceOptions &&
 											onWorkspaceChange &&
+											// isBelowMdViewport() intentionally uses the md:
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/AgentChatInput.tsx]: Fixed in 7d916ee3f. The non-dropdown AgentChatInput chrome now follows sm, including the plan-mode badge. I also widened the comment to make clear that only full-width dropdown behavior intentionally remains tied to the md breakpoint because the chat pane is still narrow in that range.

File: site/.storybook/preview.tsx [hunk 12]
```diff
@@ -50,6 +50,19 @@ export const parameters: Parameters = {
 				},
 				type: "mobile",
 			},
+			// Approximates a 1440x900 desktop viewed at 200% browser zoom,
+			// which collapses the CSS viewport to 720x450. Used by stories
+			// that verify the desktop layout still renders at common zoom
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/.storybook/preview.tsx:55]: Fixed. Reworded the viewport comment to say that below sm the page collapses into the mobile stack, so 720 px stays on the desktop branch.

File: site/src/pages/AgentsPage/AgentsPageView.tsx [hunk 13]
```diff
@@ -156,17 +156,20 @@ export const AgentsPageView: FC<AgentsPageViewProps> = ({
 	};
 
 	return (
-		<div className="flex h-full min-h-0 flex-col overflow-hidden bg-surface-primary md:flex-row">
+		<div
+			data-testid="agents-page-outer"
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/AgentsPageView.tsx]: Fixed in 7d916ee3f. Renamed the root test id to agents-page-layout and added purpose-based child selectors for the sidebar and main panels used by the stronger regression assertions.

File: site/src/pages/AgentsPage/components/AgentCreateForm.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/AgentCreateForm.tsx]: Fixed in 7d916ee3f. Removed the redundant sm:pb-4, leaving the base pb-4 to apply across breakpoints.

**Comment by jaaydenh** [comment]
Addressed the latest review feedback in 7d916ee3f and updated the PR description. Summary: aligned shell-level agents breakpoints to sm, kept only full-width dropdown behavior at md with a clarified comment, strengthened 720 px Storybook regression coverage, removed the redundant sm:pb-4, and renamed the root layout test id. Targeted Storybook tests and full site lint passed locally.

**Comment by jaaydenh** [comment]
/coder-agents-review
@codex review

File: site/src/pages/AgentsPage/components/ChatConversation/AttachmentFailureContext.tsx [hunk 14]
```diff
@@ -3,37 +3,81 @@ import {
...
-						return next;
-					});
+				getFailure: (fileId) => failures.get(fileId),
+				hasExpired: (fileId) => failures.get(fileId)?.kind === "expired",
+				markExpired: (fileId) => rememberFailure(fileId, { kind: "expired" }),
+				probeFailure: (fileId, href) => {
+					const cachedFailure = failuresRef.current.get(fileId);
+					if (cachedFailure) {
+						return Promise.resolve(cachedFailure);
+					}
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/ChatConversation/AttachmentFailureContext.tsx]: Fixed in fb7404a78. `probeFailure` now only persists `expired` results; transient `failed` probe results are returned to the caller but not cached globally, so a remount can probe again after a backend recovery. Added coverage for retrying a 502 and then caching a later 404.

File: site/src/pages/AgentsPage/components/ChatConversation/AttachmentFailureContext.tsx [hunk 15]
```diff
@@ -3,37 +3,81 @@ import {
...
-							return previous;
-						}
-						const next = new Set(previous);
-						next.add(fileId);
-						return next;
-					});
+				getFailure: (fileId) => failures.get(fileId),
+				hasExpired: (fileId) => failures.get(fileId)?.kind === "expired",
+				markExpired: (fileId) => rememberFailure(fileId, { kind: "expired" }),
+				probeFailure: (fileId, href) => {
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/ChatConversation/AttachmentFailureContext.tsx]: Added `AttachmentFailureContext.test.tsx` with `renderHook()` coverage for concurrent `probeFailure` calls. The test verifies two callers for the same file ID share one fetch and both receive the expired result.

File: site/src/pages/AgentsPage/components/ChatConversation/ExpiredFileIdsContext.tsx [hunk 16]
```diff
@@ -3,37 +3,81 @@ import {
...
 	useContext,
+	useRef,
 	useState,
 } from "react";
+import {
+	type AttachmentFailure,
+	probeAttachmentFailure,
+} from "../../utils/chatAttachments";
 
 type ExpiredFileIdsContextValue = {
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/ChatConversation/ExpiredFileIdsContext.tsx:9]: Renamed this to `AttachmentFailureContext`, `AttachmentFailureProvider`, and `useAttachmentFailure` in fb7404a78.

[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/ChatConversation/ExpiredFileIdsContext.tsx:9]: Removed `hasExpired`; text attachments now read `getFailure(fileId)` just like image attachments. The shared cache is also narrowed to persisted expired results, so transient failed probes are not hidden from future text fetch attempts.

File: site/src/pages/AgentsPage/components/ChatConversation/AttachmentBlocks.tsx [hunk 17]
```diff
@@ -469,14 +464,11 @@ const RemoteImageBlock: FC<{
...
 					setFailureState({ kind: "failed" });
 
-					void probeAttachmentFailure(href, controller.signal)
+					void probeFailure(fileId, href)
 						.then((reason) => {
 							if (!probeRequest.clear(controller)) {
 								return;
 							}
-							if (reason.kind === "expired") {
-								markExpired(fileId);
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/ChatConversation/AttachmentBlocks.tsx:478]: Removed the unreachable image-probe `isAbortError` guard. The text attachment path still keeps its abort handling because it passes an abort signal.

File: site/src/pages/AgentsPage/components/ChatTopBar.tsx [hunk 18]
```diff
@@ -103,7 +103,7 @@ export const ChatTopBar: FC<ChatTopBarProps> = ({
 					size="icon"
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/ChatTopBar.tsx:103]: Added `CollapsedSidebarZoom200DesktopWithAgent` with `agentId`, a collapsed sidebar, the `desktopZoom200` viewport, and a visible "Expand sidebar" assertion.

File: site/src/pages/AgentsPage/components/AgentChatInput.tsx [hunk 19]
```diff
@@ -1080,7 +1088,7 @@ export const AgentChatInput: FC<AgentChatInputProps> = ({
 							/>
 						)}
 						{planModeEnabled && (
-							<span className="hidden shrink-0 items-center gap-1 rounded-full bg-surface-secondary px-2 py-0.5 text-xs font-medium text-content-secondary md:inline-flex">
+							<span className="hidden shrink-0 items-center gap-1 rounded-full bg-surface-secondary px-2 py-0.5 text-xs font-medium text-content-secondary sm:inline-flex">
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/AgentChatInput.tsx:1084]: Added `desktopZoom200` viewport and Chromatic parameters to `PlanningIndicator`; the existing visible "Planning" assertion now runs at 720 px.

File: site/src/pages/AgentsPage/components/Sidebar/AgentsSidebar.tsx [hunk 20]
```diff
@@ -1138,10 +1138,10 @@ export const AgentsSidebar: FC<AgentsSidebarProps> = (props) => {
 						viewportClassName={cn(
 							"[mask-image:linear-gradient(to_bottom,transparent_0,black_20px,black_calc(100%-20px),transparent_100%)]",
 							"[-webkit-mask-image:linear-gradient(to_bottom,transparent_0,black_20px,black_calc(100%-20px),transparent_100%)]",
-							"md:[mask-image:none] md:[-webkit-mask-image:none]",
+							"sm:[mask-image:none] sm:[-webkit-mask-image:none]",
 						)}
 					>
-						<div className="flex flex-col gap-2 px-2 py-3 md:px-2">
+						<div className="flex flex-col gap-2 px-2 py-3 sm:px-2">
```
[the remark this answers is not in the document]
Comment (jaaydenh) [reply site/src/pages/AgentsPage/components/Sidebar/AgentsSidebar.tsx]: Removed the redundant `sm:px-2`; the base `px-2` remains.

**Review by DanielleMaywood — APPROVED** [review]
Assuming sufficient manual testing on viewports, LGTM :shipit:
